app/api/services/v3/Outcomes.scala

```diff
@@ -16,16 +16,16 @@
 
 package api.services.v3
 
-import api.endpoints.bfLoss.amend.v3.response.AmendBFLossResponse
-import api.endpoints.bfLoss.create.v3.response.CreateBFLossResponse
-import api.endpoints.bfLoss.list.v3.response.{ListBFLossesItem, ListBFLossesResponse}
-import api.endpoints.bfLoss.retrieve.v3.response.RetrieveBFLossResponse
-import api.endpoints.lossClaim.amendOrder.v3.response.AmendLossClaimsOrderResponse
-import api.endpoints.lossClaim.amendType.v3.response.AmendLossClaimTypeResponse
-import api.endpoints.lossClaim.create.v3.response.CreateLossClaimResponse
-import api.endpoints.lossClaim.list.v3.response.{ListLossClaimsItem, ListLossClaimsResponse}
-import api.endpoints.lossClaim.retrieve.v3.response.RetrieveLossClaimResponse
 import api.services.anyVersion.Outcomes.ServiceOutcome
+import v3.models.response.amendBFLosses.AmendBFLossResponse
+import v3.models.response.amendLossClaimType.AmendLossClaimTypeResponse
+import v3.models.response.amendLossClaimsOrder.AmendLossClaimsOrderResponse
+import v3.models.response.createBFLosses.CreateBFLossResponse
+import v3.models.response.createLossClaim.CreateLossClaimResponse
+import v3.models.response.listBFLosses.{ListBFLossesItem, ListBFLossesResponse}
+import v3.models.response.listLossClaims.{ListLossClaimsItem, ListLossClaimsResponse}
+import v3.models.response.retrieveBFLoss.RetrieveBFLossResponse
+import v3.models.response.retrieveLossClaim.RetrieveLossClaimResponse
 
 object Outcomes {
```

Comment:
There shouldn't really be any v3 or v4 packages etc below "api" anymore, should just be general-purpose stuff. "type ServiceOutcome", if it has to exist at all, could always just go under api.services.